File: engine/engine-model/src/main/scala/tools/packet/message/FindReply.scala
package tools.packet.message

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class FindReply(private var _target: String, private var _mapId: Int, private var _mapType: Int) extends PacketInput {
  def target: String = _target

  def mapId: Int = _mapId

  def mapType: Int = _mapType

  override def opcode(): SendOpcode = SendOpcode.WHISPER
}

File: engine/engine-model/src/main/java/tools/packet/message/FindReply.java
package tools.packet.message;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record FindReply(String target, int mapId, int mapType) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.WHISPER;
   }
}